cms/static/js/views/paging_header.js

```diff
@@ -74,33 +88,41 @@ define(["underscore", "backbone", "gettext", "text!templates/paging-header.under
                     start = (collection.getPageNumber() - 1) * collection.getPageSize(),
                     count = collection.size(),
                     end = start + count;
-                return interpolate('<span class="count-current-shown">%(start)s-%(end)s</span>', {
-                    start: Math.min(start + 1, end),
-                    end: end
-                }, true);
+                return HtmlUtils.interpolateHtml(
+                    HtmlUtils.HTML('<span class="count-current-shown">{start}-{end}</span>'), {
+                        start: Math.min(start + 1, end),
+                        end: end
+                    });
             },
 
             totalItemsCountLabel: function() {
                 var totalItemsLabel;
                 // Translators: turns into "25 total" to be used in other sentences, e.g. "Showing 0-9 out of 25 total".
-                totalItemsLabel = interpolate(gettext('%(totalItems)s total'), {
+                totalItemsLabel = HtmlUtils.interpolateHtml(gettext('{totalItems} total'), {
                     totalItems: this.view.collection.getTotalRecords()
                 }, true);
-                return interpolate('<span class="count-total">%(totalItemsLabel)s</span>', {
-                    totalItemsLabel: totalItemsLabel
-                }, true);
+
+                return HtmlUtils.interpolateHtml(
+                    HtmlUtils.HTML('<span class="count-total">{totalItemsLabel}</span>'), {
```

Comment:
The indentation is confusing here as it looks like the parameters are arguments to ``HtmlUtils.HTML``. I think this might be more easily read using ``HtmlUtils.joinHtml``.